runtime/queries/nix/injections.scm

```diff
@@ -129,3 +129,34 @@
   ])
   (#match? @_func "(^|\\.)writeRust(Bin)?$")
   (#set! injection.combined))
+
+;; (nixosTest) testScript
+((binding
+  ((attrpath) @_attr_name) 
```

Comment:
From the looks of other nix injection queries, `binding` has `expression` and such named fields, can you add them in as well?